### PR TITLE
Update all forum links to community.cesium.com

### PR DESCRIPTION
**Pull request by OmarShehata**
This updates all links in this repo from the old Google groups to the new [community.cesium.com](community.cesium.com) and changes names from `forum` -> `community forum` where appropriate. 

I also updated forum links in the change log to point to the right topic in the new community forum.

@shehzan10

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Ask a question
-about: Please use the forum (https://groups.google.com/forum/#!forum/cesium-dev) for general questions about using Cesium. 
+about: Please use the community forum (http://community.cesium.com/) for general questions about using Cesium.
```
Comment (mramato) [inline .github/ISSUE_TEMPLATE/question.md:3]: We should be using `https` everywhere.

File: CHANGES.md [hunk 2]
```diff
@@ -2380,7 +2380,7 @@ Beta Releases
 
 ### b28 - 2014-05-01
 
-* Breaking changes ([why so many?](https://groups.google.com/forum/#!topic/cesium-dev/CQ0wCHjJ9x4)):
+* Breaking changes ([why so many?](https://community.cesium.com/t/moving-towards-cesium-1-0/1209)):
```
Comment (shehzan10) [inline CHANGES.md:2383]: These two links don't match up.

**Comment by OmarShehata** [comment]
Fixed http -> https and the one incorrect link in changes.md.

@shehzan10

**Comment by shehzan10** [comment]
LGTM. Thanks @OmarShehata.

**Pull request merged**